packages/twilio-run/src/checks/nodejs-version.ts

```diff
@@ -1,7 +1,7 @@
 import { stripIndent } from 'common-tags';
 import { logger } from '../utils/logger';
 
-const SERVERLESS_NODE_JS_VERSION = '14.';
```

Comment:
I'm a bit torn about this one. This would be great for new projects but someone might upgrade their existing version of the Toolkit without yet upgrading to Node 16. Could you change this file by modifying `checkNodejsVersion` to check for both Node 16 or Node 14 and adjust the warning message in `printVersionWarning` to encount for both? Once Node 14 is EOL we can then remove Node 14 as possible option. But right now both are valid.

Reply:
Agree, updated in the latest version.